pgcli/packages/parseutils/meta.py

```diff
@@ -79,6 +80,8 @@ def __init__(
         self.is_aggregate = is_aggregate
         self.is_window = is_window
         self.is_set_returning = is_set_returning
+        self.is_extension = bool(is_extension)
```

Comment:
I'm having a hard time seeing how this is used.

Reply:
Fixed in https://github.com/dbcli/pgcli/pull/982/commits/ed5dc2e68ac373d69a949f13be48ab20b4d2161f.